news/user-config.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* allow user to set default configs from `.skpkgrc` or other paths.
```

Comment:
Maybe the news can be more "user"  friendly? It will be read by the community and contributors so and this is a big feature. Also pls follow the convention.